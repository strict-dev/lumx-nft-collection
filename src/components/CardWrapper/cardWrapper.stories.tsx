import { Meta, Story } from '@storybook/react'

import CardWrapper from '.'

export default {
  title: 'CardWrapper',
  component: CardWrapper
} as Meta

export const Default: Story = () => <CardWrapper>Hello</CardWrapper>
