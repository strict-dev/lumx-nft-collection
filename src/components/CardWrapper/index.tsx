type CardWrapperProps = {
  children: React.ReactNode
  className?: string
}

const CardWrapper: React.FC<CardWrapperProps> = ({ children, className }) => {
  return (
    <div className={`p-3 rounded-sm bg-neutral-800 ${className}`}>
      {children}
    </div>
  )
}

export default CardWrapper
