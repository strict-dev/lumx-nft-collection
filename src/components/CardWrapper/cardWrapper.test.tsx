import { render, screen } from '@testing-library/react'

import CardWrapper from '.'

describe('<CardWrapper />', () => {
  it('should render the heading', () => {
    const { container } = render(<CardWrapper>Hello</CardWrapper>)

    expect(container.firstChild).toMatchSnapshot()
  })
})
