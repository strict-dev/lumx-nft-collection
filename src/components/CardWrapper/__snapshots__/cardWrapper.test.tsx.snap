// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardWrapper /> should render the heading 1`] = `
<div
  class="p-3 rounded-sm bg-neutral-800 undefined"
>
  Hello
</div>
`;
